src/main/java/org/ngafid/ProcessUpload.java

```diff
@@ -113,7 +113,6 @@ public static void main(String[] arguments) throws SQLException {
      * non-existant
      * upload. This can cause the upload process to crash.
      *
-     * @param connection is the connection to the database
```

Comment:
ProcessUpload from josh/refactor currently throw errors for me, needs revisiting.